src/components/Button/Button.tsx

```diff
@@ -7,7 +7,11 @@ interface ButtonProps {
   children: React.ReactNode
   secondary?: boolean
   base?: boolean
+  /**
+   * @deprecated since 1.15.0, use accentStyle
```

Comment:
The deprecation versions I added will depend on the outcome of https://github.com/trussworks/react-uswds/pull/1045

For now I am assuming this will not be a breaking release as it was "not breaking" from USWDS' perspective,